ui/src/components/popovers/PasteModal.tsx

```diff
@@ -132,6 +167,11 @@ export default function PasteModal() {
                     )}
                   </div>
                 </div>
+		<div className="space-y-4">
+                  <p className="text-xs text-slate-600 dark:text-slate-400">
+                    Sending key codes using keyboard layout {layouts[keyboardLayout]}
```

Comment:
Nit: 
- "Sending text using keyboard layout: XXXX"
- Could you embolden the language